_posts/2025-12-15-spade-blog-aoc.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: cross-post
+title: Advent of Code Processor
+author: Frans Skarman
+date: 2025-12-15
+source: Spade Blog
+external_url: https://blog.spade-lang.org/processor-for-aoc/
+---
+
+Advent is the time of advent of code. Read how to solve day 8 in hardware with
+the help of Spade.
```

Comment:
After solving all of advent of code in hardware, day 8 became too much. The obvious solution: build a custom CPU for the problem. Find out why, and how Spade made it easy.